cumulus/parachains/runtimes/glutton/glutton-westend/src/genesis_config_presets.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! # Glutton Westend Runtime genesis config presets
+
+use crate::*;
+use alloc::vec::Vec;
+use cumulus_primitives_core::ParaId;
+use frame_support::build_struct_json_patch;
+use parachains_common::AuraId;
+use sp_genesis_builder::PresetId;
+use sp_keyring::Sr25519Keyring;
+
+/// Default value, unused in a testnet setup.
+///
+/// The parachain binary does not yet utilize genesis presets,
+/// as it must obtain the para-id from the CLI to support multiple
+/// gluttons with different para-ids connecting to a relay chain.
+/// Currently, genesis presets do not allow dynamic para-ids.
+pub const DEFAULT_GLUTTON_PARA_ID: ParaId = ParaId::new(1000);
```

Comment:
definitely not 1000 (usually, this is AssetHub),
I think there was a different number: `glutton-westend-dev-1300`
```suggestion
pub const DEFAULT_GLUTTON_PARA_ID: ParaId = ParaId::new(1300);
```